### PR TITLE
layout and css changes with doug

### PR DESCRIPTION
**Pull request by derekmhewitt**
Partner Code Review

### CODE REVIEW

**Comment by dougachrist** [comment]
thanks Derek!

File: index.html [hunk 1]
```diff
@@ -26,24 +26,26 @@
...
   <body>
     <header>
-      <div class="myName">
-        <h1><div>DOUG</div><div>CHRIST</div></h1>
-        <div class="starfish">
-          <img src="images/starfish.jpg">
-        </div>
-        <div class="secondStarfish">
-          <img src="images/starfish.jpg">
+      <div class="header_inner">
```
Comment (derekmhewitt) [inline index.html:29]: Added div w/ class of 'header_inner'

File: styles/base.css [hunk 2]
```diff
@@ -1,3 +1,7 @@
+body {
```
Comment (derekmhewitt) [inline styles/base.css:1]: Added minimum responsive style size.

File: styles/base.css [hunk 3]
```diff
@@ -38,7 +42,13 @@ header {
   z-index: 10;
 }
 
+.header_inner {
+  max-width: 688px;
```
Comment (derekmhewitt) [inline styles/base.css:46]: Confined header to max width of the body element.

File: styles/modules.css [hunk 4]
```diff
@@ -7,7 +7,7 @@ section h1 {
 
 .bottomBox li {
   display: inline;
-  margin: 10%;
+  margin: 5%;
```
Comment (derekmhewitt) [inline styles/modules.css:10]: Adjusted margin + added responsive adaptability to bottom social elements.

**Pull request merged**